### ISSUE TITLE
Game.update() failure.

### ISSUE THREAD

**Issue by cscp** [itdelatrisu/opsu#255]
**Version:** 0.15.0
**Build date:** 2017-02-05 00:04
**OS:** Windows 7 (x86)
**JRE:** 1.8.0_111
**OpenGL version:** 3.3.0 (NVIDIA Corporation)
**Stack trace:**
```
org.newdawn.slick.SlickException: Game.update() failure.
	at org.newdawn.slick.GameContainer.updateAndRender(GameContainer.java:708)
	at itdelatrisu.opsu.Container.gameLoop(Container.java:135)
	at itdelatrisu.opsu.Container.start(Container.java:79)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:216)
Caused by: java.lang.NullPointerException
	at itdelatrisu.opsu.audio.MusicController.setPitch(MusicController.java:447)
	at itdelatrisu.opsu.states.Game.enter(Game.java:1592)
	at org.newdawn.slick.state.StateBasedGame.update(StateBasedGame.java:248)
	at org.newdawn.slick.GameContainer.updateAndRender(GameContainer.java:702)
	... 3 more
```

After Trying to restart i become this::

```
Could not connect to database: 'C:\opsu!\.opsu.db'.
java.sql.SQLException: Error opening connection
	at org.sqlite.core.CoreConnection.open(CoreConnection.java:215)
	at org.sqlite.core.CoreConnection.<init>(CoreConnection.java:76)
	at org.sqlite.jdbc3.JDBC3Connection.<init>(JDBC3Connection.java:24)
	at org.sqlite.jdbc4.JDBC4Connection.<init>(JDBC4Connection.java:24)
	at org.sqlite.SQLiteConnection.<init>(SQLiteConnection.java:45)
	at org.sqlite.JDBC.createConnection(JDBC.java:114)
	at org.sqlite.JDBC.connect(JDBC.java:88)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at itdelatrisu.opsu.db.DBController.createConnection(DBController.java:65)
	at itdelatrisu.opsu.db.BeatmapDB.init(BeatmapDB.java:133)
	at itdelatrisu.opsu.db.DBController.init(DBController.java:46)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:176)
Caused by: java.lang.NullPointerException
	at org.sqlite.SQLiteJDBCLoader.extractAndLoadLibraryFile(SQLiteJDBCLoader.java:203)
	at org.sqlite.SQLiteJDBCLoader.loadSQLiteNativeLibrary(SQLiteJDBCLoader.java:334)
	at org.sqlite.SQLiteJDBCLoader.initialize(SQLiteJDBCLoader.java:64)
	at org.sqlite.core.NativeDB.load(NativeDB.java:56)
	at org.sqlite.core.CoreConnection.open(CoreConnection.java:211)
	... 12 more
```

and then i become this::

```
** Uncaught Exception! **
java.lang.RuntimeException: Resource not found: version
	at org.newdawn.slick.util.ResourceLoader.getResourceAsStream(ResourceLoader.java:69)
	at itdelatrisu.opsu.downloads.Updater.getCurrentVersion(Updater.java:195)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:186)
```

**Comment by cscp** [comment]
the game is actually unplayable because of this, please fix it so i can playtest, and have some fun!

**Comment by yugecin** [comment]
Try changing the foldername so that it doesn't have exclamation marks (!) anymore.

**Comment by cscp** [comment]
Thanks! it works now!

**Comment by itdelatrisu** [comment]
Sorry about that.  Fixed some related issues in 12f56e7 and e3a4d37 so that the game should actually tell you about the folder issue and music problems.
